<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YhcRunStatusDialog</class>
 <widget class="QDialog" name="YhcRunStatusDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/pic/blue_bg.png);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>830</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab{width:100;height:60}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>设备状态</string>
    </attribute>
    <widget class="QPushButton" name="closePushButton2">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>30</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>451</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>刮板液压泵</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_SP_HS_PUMP_FAULT_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>450</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_HS_PUMP_RUN_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>综合液压泵</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_SP_HS_PUMP_RUN_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>刮板液压泵</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_CF_FAULT_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>450</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>450</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>备用电机</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>269</x>
       <y>451</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>综合液压泵</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_HS_PUMP_FAULT_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>450</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_HS_CF_RUN_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>210</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>210</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>液压站冷却风扇</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>运行信号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>610</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>到位信号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>400</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>故障信号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>滑架推出</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_CY_OUT_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_BACKUP_EM_RUN_label">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>210</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>210</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>备用电机</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_CY_IN_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>滑架拉回</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>260</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>搭轨落下</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_CT_R_STATE_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>260</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>260</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>搭轨抬起</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_CT_D_STATE_label">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>260</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>导板推出</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>导板拉回</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_GP_OUT_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_GP_IN_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_SP_RUN_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>310</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_47">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>310</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>刮板</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_SP_FILTER_BLOCKING_label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>500</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_48">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>500</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>刮板过滤器堵塞</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_49">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>500</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>综合过滤器堵塞</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_FILTER_BLOCKING_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>500</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="YHC_OT_LEVEL_label">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>500</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>500</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>液位</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>运行数据</string>
    </attribute>
    <widget class="QLabel" name="Yhc_Oil_Pressure_label">
     <property name="geometry">
      <rect>
       <x>398</x>
       <y>40</y>
       <width>100</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/white_bg.png);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>297</x>
       <y>40</y>
       <width>100</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>综合油路压力</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Yhc_Sp_Oil_Pressure_label">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>40</y>
       <width>100</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/white_bg.png);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>100</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>刮板油路压力</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>299</x>
       <y>100</y>
       <width>100</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>液压站油位</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Yhc_Hs_Oil_Level_label">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>100</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/white_bg.png);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>100</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/pic/field_bg.png);</string>
     </property>
     <property name="text">
      <string>液压站温度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Yhc_Hs_Tempture_label">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>100</y>
       <width>100</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/pic/white_bg.png);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="closePushButton1">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>30</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
